src/main/resources/i18n/messages.properties

```diff
@@ -97,25 +97,23 @@ ranked1v1.winLossRation=Win/loss ratio
 ranked1v1.rankingFormat=#{0,number,#}
 ranked1v1.notification.title=1v1 ranked match available
 ranked1v1.notification.message=Click here to open the 1v1 tab
-teammatchmaking.leaveParty=Leave party
-teammatchmaking.title=Team matchmaking
-teammatchmaking.invitePlayer=Invite player
+teammatchmaking.leaveParty=LEAVE PARTY
+teammatchmaking.invitePlayer=INVITE PLAYERS
 teammatchmaking.invite=Invite
-teammatchmaking.availableQueues=Available queues
-teammatchmaking.playersInQueue=In Queue: {0,number,#}
-teammatchmaking.teamSize=Team size: {0,number,#}
-teammatchmaking.joinQueue=Join
-teammatchmaking.leaveQueue=Leave
-teammatchmaking.queuePopTimer={0,number,0}:{1,number,00}
-teammatchmaking.chat=Chat / Map pool?
-teammatchmaking.statistics=Statistics / Chat?
-teammatchmaking.rating=Placement (1v1)
-teammatchmaking.gameCount=Games: {0,number,#}
+teammatchmaking.playersInQueue=IN QUEUE: {0,number,#}
+teammatchmaking.queuePopTimer=STARTING: {0,number,0}:{1,number,00}
+teammatchmaking.clan=[CLAN]:
+teammatchmaking.rating=RATING: {0,number,#}
+teammatchmaking.1v1Rating=1V1 RATING: {0,number,#}
+teammatchmaking.teamRating=TEAMGAME RATING: {0,number,#}
+teammatchmaking.gameCount=GAMES PLAYED: {0,number,#}
 teammatchmaking.notification.invite.title=Invited to party
 teammatchmaking.notification.invite.message={0} has invited you to join their party
 teammatchmaking.notification.invite.join=Join
-teammatchmaking.ready=You are ready!
-teammatchmaking.notReady=Ready up!
+teammatchmaking.hint=SELECT QUEUES BELOW AND WAIT FOR MATCH
+teammatchmaking.play=PLAY FORGED ALLIANCE FOREVER
+teammatchmaking.playerTitle=PLAYING MATCHMAKING AS:
+teammatchmaking.partyTitle=YOUR PARTY:
```

Comment:
Why caps lock?

Reply:
I second this.
About merging, there arr multiple very debatable design choices I would like to discuss before merging.

Reply:
The design uses uppercase. There is a css command to style a text uppercase or lowercase, but guess what, javafx doesn't support that command. :(
We could maybe have the message in lowercase and convert it to uppercase in the controller, but I don't know if that is a better solution.

About the design choices, it is best to have this discussion on zulip together with biass.